src/CMakeLists.txt

```diff
@@ -1,11 +1,57 @@
+set(IMGUI_PATH ${CMAKE_SOURCE_DIR}/deps/imgui)
+
+add_library(imgui-backend-GL SHARED
```

Comment:
Avoid hardcoding library types in CMake. Let the client choose.

Reply:
Done.